boards/shared/Src/App/App_SharedErrorTable.c

```diff
@@ -0,0 +1,293 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include <assert.h>
+#include "App_SharedErrorTable.h"
+
+struct ErrorTable
+{
+    struct Error *errors[NUM_ERROR_IDS];
+};
+
+static void App_ResetBoardList(struct ErrorBoardList *board_list)
+{
+    for (size_t i = 0; i < NUM_ERROR_BOARDS; i++)
+    {
+        board_list->boards[i] = NUM_ERROR_BOARDS;
+    }
+
+    board_list->num_boards = 0;
+}
```

Comment:
I'm a bit confused, why do we need this? Is it mostly just something so the lists look a little nicer for debugging?

Reply:
Yeah I thought that could be nice but on a second though maybe it's no necessary. I removed this function altogether and replaced it with just `board_list->num_boards`.